libs/feature/record/src/lib/record-metadata/record-metadata.component.ts

```diff
@@ -10,8 +11,15 @@ import { map } from 'rxjs/operators'
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class RecordMetadataComponent {
-  displayMap$ = this.facade.mapApiLinks$.pipe(
-    map((links) => !!links && links.length > 0)
+  displayMap$ = combineLatest([
+    this.facade.mapApiLinks$,
+    this.facade.dataLinks$,
+  ]).pipe(
+    map(
+      ([mapLinks, dataLinks]) =>
+        (!!mapLinks && mapLinks.length > 0) ||
+        (!!dataLinks && dataLinks.length > 0)
+    )
```

Comment:
This is not ideal, eventually we'll need another type of usage like `GEODATA` or something, but for now this will do I guess

Reply:
I agree, didn't feel that well with this. I think we'll have to get back to the classification at some point. Thanks for the review!